### ISSUE TITLE
Cannot convert a block padded by 4 'u' characters. (Parameter 'index')

### ISSUE THREAD

**Issue by praaline** [UglyToad/PdfPig#782]
When parsing a 5-page PDF file, I am getting the error "Cannot convert a block padded by 4 'u' characters. (Parameter 'index')".

I used PDFill free tools to split the PDF into 5 separate files, and then the parsing worked fine (same code).

I am creating a PDFDocument with PdfDocument.Open(pdfFileName); then getting each page in a loop with document.GetPage(i + 1); and then looping over each word with page.GetWords(); I am also using the boundingBox property of each word.

Could it be related to https://github.com/UglyToad/PdfPig/issues/548 ?

**Comment by BobLd** [comment]
@praaline would you be able to share the document that's causing the issue?

**Comment by praaline** [comment]
@EliotJones yes, could you please give me an email address? We also need to know who will have access to it (let's exchange this information by email, though). Thank you

**Comment by RikanoAlienaris** [comment]
Same issue here.

![image](https://github.com/UglyToad/PdfPig/assets/52498937/814ddbb9-2773-497d-8f3f-6e5aec416e90)

The excepcion is produced when i call GetPages() in order to read the document. I can provide a PDF if it's needed.

Thank you.

**Comment by EliotJones** [comment]
This should be addressed in the latest nightly build. If not feel free to open a new issue.
